### PR TITLE
38-m3-add-pre-loaded-graphics-on-kibana: add dashboard template

### LINKED ISSUE #38: M3: add pre-loaded graphics on kibana.
To provide guidance to new users of CMCD, we need to generate pre-loaded graphics in Kibana that demonstrate the type of aggregated information that can be represented and analyzed. These pre-loaded graphics will be suggested in the project's README to help users understand the capabilities and potential of CMCD.

AC: 
- Generate a set of pre-loaded graphics in Kibana that showcase the variety of information that can be aggregated and visualized using CMCD.
- Include these pre-loaded graphics in the project's README file, providing clear instructions on how to access and interpret them.

### PR DESCRIPTION
**Pull request by adriansqualabs**

### CODE REVIEW

File: kibanaTemplate.sh
Comment (JoaquinBCh) [inline kibanaTemplate.sh:1]: I would move this inside kibana folder, what do you think?

**Review by coatesjuan — CHANGES_REQUESTED** [review]
Add execution permissions on the entrypoint.sh file so when we merge there will not be any issues in the setup of the docker

File: kibanaTemplate.sh
Comment (adriansqualabs) [reply kibanaTemplate.sh:1]: Good idea, I will move it.

**Review by coatesjuan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -148,7 +148,20 @@ http://localhost:3000/video/1/ew0KIOKAnHVybOKAnTog4oCcaHR0cHM6Ly9keGNsajl2cDNtND
 
 ## Analyze data
 
-  
+**Load dashboard template example:**
+
+There's a dashboard example, to make it available on kibana, run:
```
Comment (mcamardon) [inline README.md]: I would remove the second comma.

Comment (mcamardon) [reply README.md]: I would also put the command before the disclaimer to make it clearer.

**Review by mcamardon — APPROVED** [review]

**Pull request merged**